### PR TITLE
PWGHF: Add Selector Variables for Lc->PKPi Selector task

### PR DESCRIPTION
**Pull request by phymanshu**
- Adding selector variables in Lc->PKPi selector similar to Xic->PKPi

### CODE REVIEW

File: PWGHF/Core/SelectorCuts.h [hunk 1]
```diff
@@ -479,17 +479,17 @@ constexpr double binsPt[nBinsPt + 1] = {
...
+                                            {0.4, 0.4, 0.4, 0.4, 0., 0.005, 0., 0.005, 1., 100., 0.5, -1.},  /* 1  < pT < 2  */
+                                            {0.4, 0.4, 0.4, 0.4, 0., 0.005, 0., 0.005, 1., 100., 0.5, -1.},  /* 2  < pT < 3  */
+                                            {0.4, 0.4, 0.4, 0.4, 0., 0.005, 0., 0.005, 1., 100., 0.5, -1.},  /* 3  < pT < 4  */
+                                            {0.4, 0.4, 0.4, 0.4, 0., 0.005, 0., 0.005, 1., 100., 0.5, -1.},  /* 4  < pT < 5  */
+                                            {0.4, 0.4, 0.4, 0.4, 0., 0.005, 0., 0.005, 1., 100., 0.5, -1.},  /* 5  < pT < 6  */
+                                            {0.4, 0.4, 0.4, 0.4, 0., 0.005, 0., 0.005, 1., 100., 0.5, -1.},  /* 6  < pT < 8  */
+                                            {0.4, 0.4, 0.4, 0.4, 0., 0.005, 0., 0.005, 1., 100., 0.5, -1.},  /* 8  < pT < 12 */
+                                            {0.4, 0.4, 0.4, 0.4, 0., 0.005, 0., 0.005, 1., 100., 0.5, -1.},  /* 12 < pT < 24 */
+                                            {0.4, 0.4, 0.4, 0.4, 0., 0.005, 0., 0.005, 1., 100., 0.5, -1.}}; /* 24 < pT < 36 */
```
Comment (mfaggin) [inline PWGHF/Core/SelectorCuts.h:492]: thanks @phymanshu , ok for me to add such cuts if needed, however I'd use default values such that no new selections are applied (i.e.: I'd use `0` as default for `decLengthXY` and `normDecLXY`, and `1e+10` for `ct` and `impParXY`)

**Review by mfaggin — CHANGES_REQUESTED** [review]

File: PWGHF/TableProducer/candidateSelectorLc.cxx [hunk 2]
```diff
@@ -201,6 +201,26 @@ struct HfCandidateSelectorLc {
...
+
+    // candidate normalized decay length XY
+    if (candidate.decayLengthXYNormalised() < cuts->get(pTBin, "normDecLXY")) {
+      return false;
+    }
+
+    // candidate ct
+    if (hfHelper.ctXic(candidate) > cuts->get(pTBin, "ct")) {
+      return false;
+    }
```
Comment (mfaggin) [inline PWGHF/TableProducer/candidateSelectorLc.cxx:217]: do you really need a cut on `hfHelper.ctXic(candidate)`? If yes, I'd anyway remove it from the default selections for Lambdac, and eventually add a specific `Configurable` in the selector. I do not like the idea to have a Xic-related cut in the code of Lambdac

**Review by mfaggin — CHANGES_REQUESTED** [review]

File: PWGHF/TableProducer/candidateSelectorLc.cxx (hunk 2 above)
Comment (vkucera) [reply PWGHF/TableProducer/candidateSelectorLc.cxx]: @mfaggin Good catch!

File: PWGHF/Core/SelectorCuts.h [hunk 3]
```diff
@@ -479,17 +479,17 @@ constexpr double binsPt[nBinsPt + 1] = {
...
-                                            {0.400, 0.4, 0.4, 0.4, 0., 0.005, 0., -1.},  /* 1  < pT < 2  */
-                                            {0.400, 0.4, 0.4, 0.4, 0., 0.005, 0., -1.},  /* 2  < pT < 3  */
-                                            {0.400, 0.4, 0.4, 0.4, 0., 0.005, 0., -1.},  /* 3  < pT < 4  */
-                                            {0.400, 0.4, 0.4, 0.4, 0., 0.005, 0., -1.},  /* 4  < pT < 5  */
-                                            {0.400, 0.4, 0.4, 0.4, 0., 0.005, 0., -1.},  /* 5  < pT < 6  */
-                                            {0.400, 0.4, 0.4, 0.4, 0., 0.005, 0., -1.},  /* 6  < pT < 8  */
-                                            {0.400, 0.4, 0.4, 0.4, 0., 0.005, 0., -1.},  /* 8  < pT < 12 */
-                                            {0.400, 0.4, 0.4, 0.4, 0., 0.005, 0., -1.},  /* 12 < pT < 24 */
-                                            {0.400, 0.4, 0.4, 0.4, 0., 0.005, 0., -1.}}; /* 24 < pT < 36 */
+// default values for the cuts                m,  ptP, ptK, ptPi, chi2PCA, dL, cosp, dLXY, NdLXY, ImpParXY mass(Kpi)
```
Comment (mfaggin) [inline PWGHF/Core/SelectorCuts.h:482]:
```suggestion
// default values for the cuts                m,  ptP, ptK, ptPi, chi2PCA, dL, cosp, dLXY, NdLXY, ImpParXY, mass(Kpi)
```
super minor

**Review by mfaggin — APPROVED** [review]

**Pull request merged**